packages/Search/src/DTK_LinearBVH_decl.hpp

```diff
@@ -156,7 +156,7 @@ BoundingVolumeHierarchy<DeviceType>::BoundingVolumeHierarchy(
     // sort them along the Z-order space-filling curve
     auto permutation_indices = Details::sortObjects( morton_indices );
     Details::TreeConstruction<DeviceType>::initializeLeafNodes(
-        permutation_indices, primitives, getLeafNodes() );
+        primitives, permutation_indices, getLeafNodes() );
```

Comment:
COMMENT for consistency with other member functions that take the primitives as argument